### PR TITLE
Adding new cypress tests

### PR DESCRIPTION
**Pull request by edmoss94**
Added new cypress tests covering:

- Items in the navbar
- embedded questionnaires
- page link page items

@sheralim012 could you review to check style is consistent with current tests

### CODE REVIEW

File: cypress_tests/cypress/integration/articles/page_link_page_articles.spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+describe("Page link page article tests", () => {
```
Comment (sheralim012) [inline cypress_tests/cypress/integration/articles/page_link_page_articles.spec.js:1]: this looks good, 
and rendering page link page as section child can render section, article, questionnaires (poll, survey, quiz)
so can you add tests for these too?

File: cypress_tests/cypress/integration/footers/upper_footers.spec.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+describe("Upper footer tests", () => {
```
Comment (sheralim012) [inline cypress_tests/cypress/integration/footers/upper_footers.spec.js:1]: this can be moved to `navbar.js` file too as it's also part of site navigation

**Review by sheralim012 — CHANGES_REQUESTED** [review]

File: cypress_tests/cypress/integration/footers/upper_footers.spec.js (hunk 2 above)
Comment (edmoss94) [reply cypress_tests/cypress/integration/footers/upper_footers.spec.js]: Addressed in subsequent commit

**Comment by edmoss94** [comment]
Still scope to add more test code, but if you are happy @geoo89 suggest merging this for now and will work on adding more test coverage when I'm back from holiday next week

**Review by sheralim012 — APPROVED** [review]

**Pull request merged**